apk_info/apk_info.go

```diff
@@ -0,0 +1,38 @@
+package apk_info
+
+import (
+	"bytes"
+	"encoding/xml"
+	"fmt"
+
+	"github.com/avast/apkparser"
+)
+
+type AndroidManifest struct {
```

Comment:
As far as I see only `AndroidManifest.PackageName` is used from this struct.
Also, the struct itself is not used outside of the `GetAPKPackageName` function.
We could inline define `AndroidManifest` in the GetAPKPackageName.

Reply:
Here is my suggestion: https://github.com/bitrise-steplib/bitrise-step-android-instrumented-test/pull/1#discussion_r902452075